deploy-prod.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# This script is not currently meant to be executed from within the repository.
+# Copy or symlink it to some deployment directory.
+
+set -ex
+
+cd /var/www/dds
+
+# Update mes-aides
+git fetch origin
+git rebase origin/master
+npm install
+grunt build
+
+# Update openfisca
+npm run install-openfisca
+cat openfisca/api_config.ini | sed s/"port = 2000"/"port = 12000"/ > ~/production.ini
```

Comment:
The double quotes here are misleading, as they will be interpreted by the shell, but it looks like they are a `sed` match. I recommend you escape the whole expression: `"s/port = 2000/port = 12000/"`.